src/worker.js

```diff
@@ -182,7 +182,7 @@ class Worker {
       nextAttemptDelay: this.retryDelay,
       attemptCount: this.attempt
     }, 'Task failed, retrying')
-    this._incMonitor('ponos.finish', { result: 'task-error' })
+    this._incMonitor('ponos.finish-error', { result: 'task-error' })
```

Comment:
yea, we need this to help our data dog metrics. errors are for all errors, and finish will track only final errors or success. this will help us not double count errors